### ISSUE TITLE
[accessibility] Ensure accessible target size for default/commonly used controls

### ISSUE THREAD

**Issue by Malvoz** [openlayers/openlayers#12311]
The <span title="Web Content Accessibility Guidelines">WCAG</span> 2.1 <span title="Success Criterion">SC</span> [2.5.5 Target Size](https://www.w3.org/TR/WCAG21/#target-size) requires targets for [_pointer input_](https://www.w3.org/TR/WCAG21/#dfn-pointer-inputs) to be at least **44 by 44 in CSS pixels**. Meeting this requirement has [multiple benefits](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#benefits), it'd be great to increase the default size to pass this criteria for the commonly used/default controls such as:

- [Navigation controls](https://openlayers.org/en/latest/examples/navigation-controls.html)
- [Attribution control](https://openlayers.org/en/latest/examples/attributions.html)
- [Full screen control](https://openlayers.org/en/latest/examples/full-screen.html)
- ["Reset rotation" control](https://openlayers.org/en/latest/examples/rotation.html)
- [Overview control](https://openlayers.org/en/latest/examples/overviewmap.html)
- Popup's [Close button](https://github.com/openlayers/openlayers/issues/12308)
- Markers/[Icon symbolizers](https://openlayers.org/en/latest/examples/icon.html)(?)
- [Custom controls](https://openlayers.org/en/latest/examples/custom-controls.html)(?)

**Comment by Malvoz** [comment]
Accessibility issues don't go away this easily, Mr. stale bot.

**Comment by ahocevar** [comment]
Thanks for reporting this and other accessibility issues, @Malvoz. It would be easier to make them go away if we had pull requests which implement the changes you suggest.

I guess many of the issues can be solved by css changes, and some require changes in the markup. For the css ones, I'd imagine a second css file, `ol-accessibility.css`, which overrides the standard css to solve accessibility issues. This should be straightforward and not require much discussion. For markup changes, some discussion ahead of the implementation might be necessary, like in #12485. But the also for these issues, a pull requests would be the best way to get them resolved.

So the question is: would you be able to provide such pull requests? And if not, who would be?

**Comment by Malvoz** [comment]
Thanks for the respose @ahocevar. I don't quite have time to get familiar with each library (I've reported 17 out of [36 identified issues with Leaflet](https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/247), and have 15+ issues to report to both MapBox and MapLibre GL JS, and others). I think people who are already familiar with the code and use OpenLayers themselves are better candidates, and as you say these are mostly simple changes.
